--- som_pak.txt	2013-05-14 11:40:25.328464050 +0200
+++ som_pak.txt2	2013-05-14 11:57:00.918485299 +0200
@@ -261,8 +261,8 @@
  	  /* if the row in the input file starts 
  	     with # or with newline, skip it     */
  	  if (strcmp(s,"")==NULL || s[0]=='#')
---- mapinit.c	1995-04-06 17:01:00.000000000 -0700
-+++ mapinit.c	2012-04-16 20:18:28.000000000 -0700
+--- som_pak-3.1/mapinit.c	1995-04-06 17:01:00.000000000 -0700
++++ som_pak-3.1/mapinit.c	2012-04-16 20:18:28.000000000 -0700
 @@ -68,7 +68,7 @@
        exit(0);
      }
